chart/config/proxy/lib.https_redirect.conf

```diff
@@ -12,3 +12,13 @@ if ($http_x_glb_forwarded = 'true' ) {
 if ($https_test = '+nhttps+gclb') {
     return 308 https://$host$request_uri;
 }
+
+if ($proxy_protocol_server_port != '443') {
```

Comment:
I realize this port 443 and also 80 need to be parametrized when using the proxy protocol as those ports do not guarantee TLS or non-TLS by themselves but the operator of the service would know which is the secure port and which one is the plaintext and they can set that in the help chart (when turning on proxy protocol)